src/Hazelcast.Net/Networking/RoutingModes.cs

```diff
@@ -36,7 +31,12 @@ public enum RoutingModes : byte
         /// <para>Note that it however does not guarantee that the operation will always be
         /// executed on the owner, as the member table is only updated every 10 seconds.</para>
         /// </summary>
-        AllMembers = 2
+        AllMembers = 1,
```

Comment:
is the existing release broken due to this mismatch? Will the fix be backported?

Reply:
Yes, this was the reason. I'm not sure about the backport. Since it's a cosmetic issue, another release may not be needed.